scripts/report-code-metrics.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+set -e
+
+CURRENT_TIME=$(date +%s)
+
+# This script calculates code quality metrics.
+# These are evaluated in the daily travis build and reported to datadog.
+# Other metrics are computed in the management command `report_code_metrics`
+
+function send_metric_to_datadog() {
```

Comment:
this seems to be a copy of what we have in the main test script. Can we put it in a utils file?

Reply:
I initially thought I'd change this more (I dropped the `test` and `partition` tags), but agree that not duplicating this would be better.  Thoughts on how to do that?  I guess a `scripts/datadog-utils.sh` that I can source from both places?

Reply:
yes, that seems fine to me.